### PR TITLE
[14.0][FIX] pos_sale_order_load: fix bug

### PR DESCRIPTION
**Pull request by dessanhemrayev**
The part of the code where the _loadPartners method is called has been deleted

According to this [review](https://github.com/OCA/pos/pull/1142#discussion_r1503127806)

### CODE REVIEW

File: pos_sale_order_load/static/src/js/OrderManagementScreen/SaleOrderManagementScreen.js [hunk 1]
```diff
@@ -104,24 +104,7 @@ odoo.define("pos_sale_order_load.SaleOrderManagementScreen", function (require)
                 );
                 if (order_partner) {
```
Comment (aayartsev) [inline pos_sale_order_load/static/src/js/OrderManagementScreen/SaleOrderManagementScreen.js:105]: What will happen if ther is no order_partner ?

**Review by aayartsev — CHANGES_REQUESTED** [review]
Check please

**Review by aayartsev — APPROVED** [review]
Looks good.

**Comment by francesco-ooops** [comment]
@vuwnevska better now?

**Review by vuwnevska — COMMENTED** [review]
on line 100
you can use 
`await this.env.pos.load_new_partners(sale_order.partner_id[0]);`
as we already know the partner we need only this one, loading all new on big db can take a lot of time

**Review by vuwnevska — APPROVED** [review]

**Comment by francesco-ooops** [comment]
@legalsylvain good for you?

**Pull request merged**